gfxr_ext/decode/dive_annotation_processor.h

```diff
@@ -59,6 +60,12 @@ class DiveAnnotationProcessor : public gfxrecon::decode::AnnotationHandler
         std::string           name = "";
     };
 
+    struct DrawCallCounts
+    {
+        std::vector<uint64_t> begin_command_buffer_draw_call_counts = {};
```

Comment:
sorry i still dont quite get why begin_command_buffer_draw_call_counts is a vector. m_draw_call_counts_map is a hashmap with cmd_handle as the key, so DrawCallCounts is corresponding to 1 cmd buffer. and if this is the case, each cmd buffer should have only 1 count?

Reply:
Updated. Originally had as vector due to previous iteration where there was a bug in the tree view.